aqt/cli.py

```diff
@@ -27,6 +27,7 @@
 import os
 import platform
 import sys
+from packaging.version import Version, parse
```

Comment:
packaging seems a known third party library.
It is better to add it to tox.ini in section isort;
```
[isort]
known_first_party = aqt
known_third_party = docutils,flake8,pyannotate_runtime,pytest,pytz,setuptools,sphinx,yaml
```
and change import  order,
```
 import os
 import platform
 import sys
+
+import yaml
 from packaging.version import Version, parse
-
-import yaml
```